dev_tools/install_macos.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+
+    # Install some custom requirements on macOS
+    # e.g. brew install pyenv-virtualenv
+    export PYTHON=3.7.0
+    brew update
+    brew install openssl readline pyenv-virtualenv
+    brew outdated pyenv || brew upgrade pyenv
+    pyenv install $PYTHON
+    export PYENV_VERSION=$PYTHON
```

Comment:
Is it possible to pass this as a flag instead of via an exported variable?

Reply:
Why would we want to pass it in as a flag if it's always set to `$PYTHON`?

Reply:
I wanted to avoid setting `$PYTHON`.